### PR TITLE
Add a layer opacity example

### PR DESCRIPTION
**Pull request by tschaub**
This adds an example showing how `layer.setOpacity()` can be used to adjust the layer opacity.

### CODE REVIEW

File: examples/layer-opacity.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+tags: "opacity"
+cloak:
+  - key: get_your_own_D6rA4zTHduk6KOKTXzGB
+    value: Get your own API key at https://www.maptiler.com/cloud/
+---
+<div id="map" class="map"></div>
+<label>
+  Layer opacity
+  <input id="opacity-input" type="range" min="0" max="1" step="0.01" value="1" />
+  <span id="opacity-output"></span>
```
Comment (marcjansen) [inline examples/layer-opacity.html:16]:
```suggestion
  <span id="opacity-output">1.00</span>
```

**Review by marcjansen — APPROVED** [review]
Thanks. Initially the opacity isn't displayed, which is super minor. Please merge with or without taking the suggestion.

File: examples/layer-opacity.html (hunk 1 above)
Comment (tschaub) [reply examples/layer-opacity.html:16]: Thanks, @marcjansen.  I was bugged by the same and added a call to `update` in the latest commit.

Comment (marcjansen) [reply examples/layer-opacity.html:16]: Same effect and fine for me.

**Pull request merged**